test/e2e/framework/util.go

```diff
@@ -1247,7 +1249,7 @@ func waitTimeoutForPodNoLongerRunningInNamespace(c *client.Client, podName strin
 }
 
 func waitTimeoutForPodReadyInNamespace(c *client.Client, podName string, namespace string, timeout time.Duration) error {
-	w, err := c.Pods(namespace).Watch(api.SingleObject(api.ObjectMeta{Name: podName}))
+	w, err := c.Pods(namespace).Watch(api.SingleObject(api.ObjectMeta{Name: podName, ResourceVersion: "1"}))
```

Comment:
Why is this necessary?  If we omit the resource version it should fetch the current state and then return it